app/src/main/res/layout/settings_screen_fxa_profile.xml

```diff
@@ -2,7 +2,9 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<!-- TODO comment why this is a separate file -->
+<!-- This file is currently extremely similar to settings_screen_buttons. However, it is expected
```

Comment:
I think you quickly make this less verbose by removing some qualifiers:

> This file is similar to settings_screen_buttons. However, it is expected to expand as more FxA options are added. This is why we keep it as a separate file instead of handling both in the same file.

My way to quickly take long comments and make them short is just removing qualifiers.

I like to keep comments briefly because people might actually read them then. 🤣